.github/workflows/native.yml

```diff
@@ -41,10 +44,37 @@ jobs:
         uses: ilammy/msvc-dev-cmd@v1
         if: ${{ matrix.os == 'windows-2019' }}
       - name: Build
+        if: ${{ matrix.uploaded_filename != 'sbtn-aarch64-pc-linux' }}
         shell: bash
         run: |
           echo $(pwd)
           sbt clean nativeImage
+      - name: Build Linux aarch64
+        if: ${{ matrix.uploaded_filename == 'sbtn-aarch64-pc-linux' }}
+        uses: uraimo/run-on-arch-action@v2.5.0
+        with:
+          arch: aarch64
+          distro: ubuntu22.04
+
+          # Speeds up builds per the run-on-arch-action README
+          githubToken: ${{ github.token }}
+
+          # The shell to run commands with in the container
+          shell: /bin/bash
+
+          # build-essential and libz-dev are required to build native images.
+          install: |
+            apt-get update -q -y
+            apt-get install -q -y curl openjdk-8-jdk build-essential libz-dev
+            # Does not work, see https://github.com/sbt/sbt/issues/6614
+            #echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | tee /etc/apt/sources.list.d/sbt.list
+            #curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | apt-key add
+            #apt-get update -q && apt-get install -q -y sbt
+            # Workaround:
+            curl -Ls https://scala.jfrog.io/artifactory/debian/sbt-1.8.0.deb -o sbt-1.8.0.deb
+            apt install -y ./sbt-1.8.0.deb
```

Comment:
Another alternative to install `sbt` inside this QEMU virtual machine would be to use sdkman:
```sh
curl -s "https://get.sdkman.io" | bash
source "/root/.sdkman/bin/sdkman-init.sh"
sdk install sbt
```
We could also use coursier, like seen in the scala-ci or coursier repo... However I didn't do that since coursier does not publish a aarch64 64k page-size compatible binary yet (but soon :wink:), so I couldn't test that locally on my machine... (It should work in the CI virtual machine though)